### PR TITLE
Added the indel and base_recal steps from rare to rna, fixes #468

### LINKED ISSUE #468: Indel and base recalibration module RNA seq

### PR DESCRIPTION
**Pull request by jemten**

### CODE REVIEW

File: lib/MIP/Recipes/Pipeline/Rna.pm [hunk 1]
```diff
@@ -284,6 +287,73 @@ sub pipeline_rna {
...
+            my $insample_directory =
+              catdir( $active_parameter_href->{outdata_dir},
+                $sample_id, $active_parameter_href->{outaligner_dir} );
+            my $outsample_directory =
+              catdir( $active_parameter_href->{outdata_dir},
+                $sample_id, $active_parameter_href->{outaligner_dir} );
+
+            analysis_gatk_realigner(
+                {
+                    parameter_href          => $parameter_href,
```
Comment (henrikstranneheim) [inline lib/MIP/Recipes/Pipeline/Rna.pm]: Order AB

File: lib/MIP/Recipes/Pipeline/Rna.pm [hunk 2]
```diff
@@ -284,6 +287,73 @@ sub pipeline_rna {
...
+            my $insample_directory =
+              catdir( $active_parameter_href->{outdata_dir},
+                $sample_id, $active_parameter_href->{outaligner_dir} );
+            my $outsample_directory =
+              catdir( $active_parameter_href->{outdata_dir},
+                $sample_id, $active_parameter_href->{outaligner_dir} );
+
+            analysis_gatk_baserecalibration(
+                {
+                    parameter_href          => $parameter_href,
```
Comment (henrikstranneheim) [inline lib/MIP/Recipes/Pipeline/Rna.pm]: Order AB

**Comment by jemten** [comment]
This pull request got a bit messy. However the parameters have been ordered alphabetically

**Comment by henrikstranneheim** [comment]
👍

**Pull request merged**